scripts/docsgen/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/coder/terraform-provider-coder/provider"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// This script patches Markdown docs generated by `terraform-plugin-docs` to expose the original deprecation message.
+
+var (
+	reDeprecatedProperty = regexp.MustCompile("`([^`]+)` \\(([^,\\)]+), Deprecated\\) ([^\n]+)")
```

Comment:
readability: use backticks for regexes so you don't need to escape all the things

Reply:
no so easy here? we have backticks in the regexp already, and we can't escape them

Reply:
🤦 never mind then!